page-changelog.php

```diff
@@ -59,7 +59,6 @@
 MD5: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.md5">owncloud-9.0.10.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.md5">owncloud-9.0.10.zip.md5</a></br>
 SHA256: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.sha256">owncloud-9.0.10.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.sha256">owncloud-9.0.10.zip.sha256</a></br>
 PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.asc">owncloud-9.0.10.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.asc">owncloud-9.0.10.zip.asc</a></br>
-Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/9.0/owncloud">9.0 Release Channel</a> and <a href="https://download.owncloud.org/download/repositories/stable/owncloud/">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
```

Comment:
That was one line killed to much 😄  Its available in the 9.0 release channel at:

https://download.owncloud.org/download/repositories/9.0/owncloud

but not at:

https://download.owncloud.org/download/repositories/stable/owncloud

so only the part of the stable release channel needs to be killed.